cartography/intel/aws/ecr.py

```diff
@@ -28,15 +28,16 @@ def get_ecr_repositories(boto3_session: boto3.session.Session, region: str) -> L
     return ecr_repositories
 
 
-@timeit
-@aws_handle_regions
```

Comment:
Probably still want the `@aws_handle_regions` decorator here for the error handling I think